src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -180,4 +180,26 @@ public function getFlashes($types = null)
 
         return $result;
     }
+
+    /**
+     * Return a environment variable (defined in .env for example).
+     *
+     * If the $name is null or empty, null is returned.
+     *
+     * @return string|null
+     */
+    public function getEnv($name = null)
+    {
+        if (null === $name || '' === $name) {
+            return null;
+        }
+
+        try {
+            $request = $this->getRequest();
+        } catch (\RuntimeException $exception) {
+            return null;
```

Comment:
I would suggest not to silence this exception and instead make `throw new \RuntimeException('The "app.env" variable is not available.', 0, $exception);` ?

Reply:
or verify if `null === $this->requestStack` directly.

Reply:
Ok, I've just added a test for the presence of the `RequestStack`.